src/test/lib/utils/plugin_manager_test.cpp

```diff
@@ -69,6 +69,83 @@ TEST_F(PluginManagerTest, LoadPluginAutomaticUnload) {
   EXPECT_FALSE(sm.has_table("DummyTable"));
 }
 
+TEST_F(PluginManagerTest, LoadingUnloadingUserExecutableFunctions) {
+  auto& pm = Hyrise::get().plugin_manager;
+  auto& plugins = get_plugins();
+
+  EXPECT_EQ(plugins.size(), 0u);
+  pm.load_plugin(build_dylib_path("libhyriseTestPlugin"));
+  pm.load_plugin(build_dylib_path("libhyriseSecondTestPlugin"));
+  EXPECT_EQ(plugins.size(), 2u);
+
+  {
+    auto user_executable_functions = pm.user_executable_functions();
+
+    EXPECT_EQ(user_executable_functions.size(), 3);
+    EXPECT_EQ(user_executable_functions.count({"hyriseSecondTestPlugin", "OurFreelyChoosableFunctionName"}), 1);
```

Comment:
3* `.contains()`? clang-tidy (versions 13+) might complain.

Reply:
Makes sense. I think some time ago I always wanted to use contains but it was unsupported. Now it is supported and I don't use it :)

Reply:
I fixed some other occurrences along the way.